scripts/qa_crowbarsetup.sh

```diff
@@ -3890,6 +3890,7 @@ function oncontroller_manila_generic_driver_setup()
         openstack role add --project manila-service --user admin admin
     fi
     export OS_TENANT_NAME='manila-service'
+    export OS_PROJECT_NAME=$OS_TENANT_NAME
```

Comment:
in v3 openrc I see things like
```
export OS_PROJECT_ID=b6833c5cb75c479899d159b7b0f49856
export OS_PROJECT_NAME="cloud"
export OS_USER_DOMAIN_NAME="Default"
if [ -z "$OS_USER_DOMAIN_NAME" ]; then unset OS_USER_DOMAIN_NAME; fi

# unset v2.0 items in case set
unset OS_TENANT_ID
unset OS_TENANT_NAME
```
and other way round for v2
so could it disturb things if both are set?